src/backend/distributed/metadata/metadata_cache.c

```diff
@@ -189,12 +205,10 @@ static ScanKeyData DistObjectScanKey[3];
 
 /* local function forward declarations */
 static bool IsCitusTableViaCatalog(Oid relationId);
-static ShardIdIndexSlot * LookupShardIdIndexSlot(ShardIdIndexSlot *table,
-												 size_t tableSize, uint64 shardId);
-static CitusTableCacheEntry * LookupShardIndex(int64 shardId, int *shardIndex);
+static ShardIdCacheEntry * LookupShardIdCacheEntry(int64 shardId);
 static CitusTableCacheEntry * LookupCitusTableCacheEntry(Oid relationId);
```

Comment:
seems we have duplicate declaration for `LookupCitusTableCacheEntry`

Reply:
```suggestion
```